build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
```

Comment:
Did we make the choice of not setting e.g. the `errexit` and `pipefail` options, and if so why?

Reply:
I initially restricted myself to what was available in /bin/sh on each platform, but I may just change this to use bash.

Reply:
OK, I'm not familiar with what platform ships what shell by default, so I'll let you determine whether relying on bash is OK.

Reply:
I realized that we don't want `errexit` set, since we want to capture the error codes and log them as part of the script. We don't use pipelines in this script, but in case we ever add them, I've added `pipefail`.